api/db/backup.js

```diff
@@ -0,0 +1,47 @@
+const exec = require('child_process').exec;
+
+// Retrieve database connection parameters
+const getDbUrl = () => {
+  require('dotenv').config();
+  return process.env.DB_URL;
+};
+
+const createBackup = () => {
+  // Prepare database archive filename
+  currentDate = new Date();
+
+  let databaseDumpFilename =
+    'tisn_db_dump_' +
+    currentDate.getFullYear() +
+    '_' +
+    (currentDate.getMonth() + 1) +
+    '_' +
+    currentDate.getDate() +
+    '.gz';
+
+  // Prepare mongodump command
+  let dumpCmd =
```

Comment:
Could you elaborate on how you constructed the command? I have to check it when I get back home, but I think I use some different and/or extra arguments.

Also, same here regarding `const` instead of `let` and the use of template literals/strings.